apiserver/grpc/server.go

```diff
@@ -58,11 +58,15 @@ func (g *GRPCServer) ApplyOneway(ctx context.Context, req *proto.ApplyPistageReq
 	if err != nil {
 		return nil, err
 	}
+
+	ctx, cancel := context.WithTimeout(ctx, g.timeout)
+	defer cancel()
```

Comment:
Actually cannot defer cancel here ( and cannot use the ctx passed into this func ) because it will cause the task never be executed. ( may need to test? )

Reply:
Yes I think I accidentally committed some random changes I was trying, will revert the context-related changes